lib/ecto/type.ex

```diff
@@ -828,7 +828,6 @@ defmodule Ecto.Type do
   defp cast_decimal(term) when is_binary(term) do
     case Decimal.parse(term) do
       {:ok, decimal} -> check_decimal(decimal, false)
-      {decimal, ""} -> check_decimal(decimal, false)
```

Comment:
This is a forward compatibility layer for `Decimal` 2.0

Reply:
Oh, I checked only version 1.x, restored